ui/src/queries/useUmapData.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query';
+
+export const useUmap = () => {
```

Comment:
More descriptive name? `usePrefitUmapEmbedding` ? Im not sure what else... some ideas:
- `useBedbaseUmap`
- `useUmapEmbedding`
- `usePretrainedUmap`